tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/CDITests.java

```diff
@@ -57,15 +55,19 @@
             }
         };
 
-        List<Object[]> tests = new ArrayList<>();
+        return Stream.of(
+            // -- Weld --
+            Arguments.of("weld", "cdi2", null),
+            Arguments.of("weld", "cdi-spi", null), // Weld >= 3.1.2
+            Arguments.of("weld", "decorate", null), // Weld >= 3.1.2
+            // Arguments.of("weld", "cdi-decorate", null), // Not supported
 
-        tests.add(new Object[]{"weld", "cdi-spi", null}); // Requires Weld >= 3.1.2
-        tests.add(new Object[]{"weld", "cdi2", null});
-        tests.add(new Object[]{"weld", "decorate", null}); // Requires Weld >= 3.1.2
-        tests.add(new Object[]{"owb", "cdi-spi", removeJettyWebXml});
-        tests.add(new Object[]{"owb", "cdi2", null});
-        // tests.add(new Object[]{"owb", "decorate", null});  // Will not be supported
-        return tests.stream().map(Arguments::of);
+            // -- Apache OpenWebBeans --
+            Arguments.of("owb", "cdi-spi", removeJettyWebXml),
+            Arguments.of("owb", "cdi2", null)
+            // Arguments.of("owb", "decorate", null), // Not supported
+            // Arguments.of("owb", "cdi-decorate", null) // Not supported
+        );
```

Comment:
I went ahead and cleaned up the test Arguments (less verbose boilerplate, sectioned off with comments)

Reply:
You changed the meaning of the comments just a little bit.  Fixing...